src/pyspextool/setup_utils.py

```diff
@@ -330,15 +333,15 @@ def set_instrument(instrument_name):
 
     check_file(bad_pixel_mask_file)
 
-    setup.state['raw_bad_pixel_mask'] = fits.getdata(bad_pixel_mask_file)
+    # setup.state['raw_bad_pixel_mask'] = fits.getdata(bad_pixel_mask_file)
+    # TODO: Figure out how to get this to work on github tests
```

Comment:
@michaelcushing , `fits.getdata` doesn't seem to work well on the github testing server. It's probably something to do with the way it caches the file. Is it possible to load the actual data when it's about to be used rather than during the setup? 

I am also posting in the Astropy Slack to see if there's a way to tell `fits.getdata` to follow the pointer.

Reply:
Astropy Slack helped me fix this so it's all good!